### PR TITLE
[#609] Changed in mailview system for different language users (connects #609) #617

### PR DESCRIPTION
**Pull request by razu9861**

### CODE REVIEW

File: app/MyApp/app/Home-Urdu.css [hunk 1]
```diff
@@ -571,39 +558,51 @@ body{
     width: 233px;
     font-weight: 900;
 }
-
 .login-form > .bbf-form{
-
+
```
Comment (dogi) [inline app/MyApp/app/Home-Urdu.css]: please remove this space character

File: app/MyApp/app/collections/Members.js [hunk 2]
```diff
@@ -6,8 +6,8 @@ $(function() {
 			if (this.login) {
 				return App.Server + '/members/_design/bell/_view/MembersByLogin?include_docs=true&key="' + this.login + '"'
 			} else if (this.manager) {
-               			return App.Server + '/members/_design/bell/_view/managerMembers?include_docs=true'
-		        } else if (this.skip) {
+               	return App.Server + '/members/_design/bell/_view/managerMembers?include_docs=true'
```
Comment (dogi) [inline app/MyApp/app/collections/Members.js:7]: both next line need an other indent

File: app/MyApp/app/views/MailView.js [hunk 3]
```diff
@@ -86,26 +86,68 @@ $(function() {
...
                         return
                     }
                 });
+                var languageDictValue;
+                var lang = getLanguage(mmodel.get('_id'))
+                languageDict = getSpecificLanguage(lang);
+                var temp
+                var that = this
+                var currentdate = new Date();
+                    var mailBody = languageDict.attributes.Hi+ '&nbsp;' + ' <b>' + username + '</b>' +',<br>' + '<br>' + languageDict.attributes.Your_Request_Has_Been_Accepted + '<br>';
```
Comment (dogi) [inline app/MyApp/app/views/MailView.js]: this and next 10 lines have an indent too much

File: app/MyApp/app/views/MailView.js [hunk 4]
```diff
@@ -86,26 +86,68 @@ $(function() {
...
+                var body = App.languageDict.attributes.Hi + '&nbsp;' + '<b>'+ $.cookie('Member.login') +  '</b>' + ',<br>' + '<br>' + App.languageDict.attributes.Member + ' <b>' + username + '</b> ' + App.languageDict.attributes.Has_Requested_Promote + '<br/><br/>';
+                mailView.updateMailBody(body) 
+                var languageDictValue;
+                var lang = getLanguage(mmodel.get("_id"))
+                languageDict = getSpecificLanguage(lang);
+                var temp
+                var that = this
+                var currentdate = new Date();
+                var mailBody = languageDict.attributes.Hi+ '&nbsp;' + '<b>' + username + '</b>'+ ',<br>' + '<br>' + languageDict.attributes.Your_Request_Has_Been_Rejected;
+                    temp = new App.Models.Mail()
```
Comment (dogi) [inline app/MyApp/app/views/MailView.js]: also this and next ~10 lines have an indent too much

File: app/MyApp/app/views/MemberForm.js [hunk 5]
```diff
@@ -131,24 +129,26 @@ $(function() {
...
-                temp.set("body", mailBody)
-                temp.set("sendDate", currentdate)
-                temp.set("entityId", $.cookie('Member._id'))
-                temp.save()
+                var languageDictValue;
+                var lang = getLanguage(m.get("_id"))
+                languageDict = getSpecificLanguage(lang);
+                var mailBody = languageDict.attributes.Hi + '&nbsp;' + '<b>'+ m.get("login")+  '</b>' + ',<br>' + '<br>' + languageDict.attributes.Member + ' <b>' + $.cookie('Member.login') + '</b> ' + languageDict.attributes.Has_Requested_Promote
+                + '<br/><br/><button class="btn btn-primary" id="promote-accept" value="' + $.cookie('Member._id') + '" >'+languageDict.attributes.Accept+'</button>&nbsp;&nbsp;<button class="btn btn-danger" id="promote-reject" value="' + $.cookie('Member._id') + '" >'+languageDict.attributes.Reject+'</button>';
+                    temp = new App.Models.Mail()
```
Comment (dogi) [inline app/MyApp/app/views/MemberForm.js:142]: also here and next ~9 lines have an indent too much

File: app/MyApp/index.html [hunk 6]
```diff
@@ -1508,15 +1508,16 @@ <h3 class="bigheader"><div style="width:95%"><%= languageDict.attributes.Step%>-
...
                 <div style="width: 49%;padding: 12px;background: white;position: absolute;margin-top: 6px;margin-left: 10px;border: solid 1px;border-radius: 11px;"> &nbsp; <%= languageDict.attributes.Sender %> &nbsp;<%= firstName %>&nbsp;<%= lastName %> &nbsp; (<%= email %>)  </div>
                 <div id="mail-body" style="background: white;margin-top: 56px;position: absolute;margin-left: 9px;width: 50%;height: 231px;border: solid 1px;border-radius: 11px;padding-left:12px;"><br/><%=body%></div>
             </div>
+            <div style="margin-left: 36%;margin-top: 20px;position: absolute;"><button class="btn btn-info" id="backpage" ><%= languageDict.attributes.Back %></button>&nbsp;&nbsp;<button class="btn deleteBtn btn-danger" value="<%= modelNo %>"  ><%= languageDict.attributes.DeleteLabel %></button>&nbsp;&nbsp;<button class="btn btn-primary" onclick="showComposePopup()" id="replyMailButton"><%= languageDict.attributes.Reply %></button><% if(isAttachment==1){ %>&nbsp;&nbsp<a role='button' class='btn btn-primary' href=<%= attchmentURL %> target="_blank"><%= languageDict.attributes.Attachment %></a><%}%></div>
```
Comment (dogi) [inline app/MyApp/index.html]: `@nbsp` needs `;` on the end

**Review by dogi — CHANGES_REQUESTED** [review]

File: app/MyApp/app/Home-Urdu.css (hunk 1 above)
Comment (Pr0chin) [reply app/MyApp/app/Home-Urdu.css]: white spaces should be removed.. should be in proper structure

File: app/MyApp/app/Home-Urdu.css [hunk 7]
```diff
@@ -1150,9 +1150,9 @@ textarea[name="comment"]
     margin-left: 20px;
     width:auto;
 }
-.form .btn {
+/*.form .btn {
     margin: 5px 20px 20px 0px;
-}
+} */
```
Comment (Pr0chin) [inline app/MyApp/app/Home-Urdu.css:1154]: +/*.form .btn {
      margin: 5px 20px 20px 0px;
 -}
 +} */
comment should be removed

**Review by Rupesh87 — APPROVED** [review]

**Review by Pr0chin — APPROVED** [review]

**Pull request closed without merging**